api-tests/src/steps/cri-steps.ts

```diff
@@ -289,5 +290,9 @@ When(
       this.userId,
       generatePostVcsBody(credentials),
     );
+    for (const credential of credentials) {
+      await cimitStubClient.postDetectCi({ signed_jwt: credential });
```

Comment:
I assume this is fairly speedy - we could try and skip if we know there's no CI, but this is a better reflection of core behaviour :shrug:

Reply:
Yeah, I did think about that but then decided not to. For the amount we're actually using this step I don't think it's a going to be a massive timesaver.